### PR TITLE
FEAT: added products page without functionality.

### PR DESCRIPTION
**Pull request by zoyatanweer**
- Added Products page (without functionality)
- Added routes to header component 

Files to check: 

- [ ] components/products/Products.jsx
- [ ] components/products/FilterSection.jsx
- [ ] src/data/data.jsx
- [ ] components/header/Header.jsx

### CODE REVIEW

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -13,17 +14,19 @@ const Header = () => {
     <nav className="navigation-container">
       <div className="nav-brand">Falcon Store</div>
       <li className="list-item list-non-bullet">
-        <a className="link link-active" href="/index.html">
+        {/* <a className="link link-active" href="/index.html">
```
Comment (rutu14) [inline src/components/header/Header.jsx]: Suggestion: Can remove this comment, if no longer needed.

File: src/components/products/Products.jsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                    </button>
+                    {/* <button className="heart-btn">
+                      <i className="far fa-heart"></i>
+                    </button> */}
+                    <button className="heart-btn">
+                      {<AddToWishlist className="far fa-heart" />}
+                      {/* {<FontAwesomeIcon icon="fa-regular fa-heart" />} */}
+                    </button>
+                    <button className="share-btn">
+                      {/* <i className="fas fa-share-alt"></i> */}
```
Comment (rutu14) [inline src/components/products/Products.jsx]: Suggestion: Can remove this comment, if no longer needed.

File: src/components/products/Products.jsx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                    {hasDiscount && (
+                      <span className="striken-text"> ₹{originalPrice}</span>
+                    )}
+                  </p>
+                  <div className="card-container-action">
+                    <button className="btn-cart">
+                      <i className="fas fa-shopping-cart"></i>Add to cart
+                    </button>
+                    {/* <button className="heart-btn">
+                      <i className="far fa-heart"></i>
```
Comment (rutu14) [inline src/components/products/Products.jsx]: Suggestion: Can remove this comment, if no longer needed.

**Review by rutu14 — APPROVED** [review]
The code structure is good to merge.

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks amazing. Good to merge.

File: src/components/products/FilterSection.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+// import React from "react";
```
Comment (abhishektanwar) [inline src/components/products/FilterSection.jsx]: Why is this file a jsx file? We are not returning any html from it.

File: src/components/products/Products.css [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+}
+.filter-radio,
+.filter-checkbox {
+  margin-right: 10px;
+}
+
+/* products-grid */
+.grid-3-column-layout {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```
Comment (abhishektanwar) [inline src/components/products/Products.css:64]: here you could use something like `grid-template-column:repeat(5,280px)` to make it more responsive with other required css

File: src/components/products/Products.css [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+  position: absolute;
+  left: 0rem;
+  top: 0rem;
+}
+.non-striken {
+  text-decoration: none;
+}
+.card-rating {
+  /* background-color: #fff0f3; */
+  /* background-color: #ffccd5; */
```
Comment (abhishektanwar) [inline src/components/products/Products.css]: remove comments if not required

**Review by abhishektanwar — APPROVED** [review]
Code is clean and understandable .UI also looks amazing . 
Good to merge

**Comment by zoyatanweer** [comment]
Thankyou guys for all your reviews. I have made the suggested changes.

**Pull request merged**